elfo-configurer/src/lib.rs

```diff
@@ -244,7 +243,6 @@ impl Configurer {
                 Ok(Ok(_)) | Err(_) => None,
                 Ok(Err(reject)) => Some((group, reject.reason)),
             })
-            // TODO: provide more info.
```

Comment:
Just wanted to ask, what kind of info do you think we should provide here? I am fine to keep the comment, just want it to be more specific.

Reply:
Actor keys